### ISSUE TITLE
MSN Error 

### ISSUE THREAD

**Issue by pro7777777777** [farshadz1997/Microsoft-Rewards-bot#104]
Hello, everything is working fine except for the msn games it keeps saying that it failed to complete them

**Comment by farshadz1997** [comment]
Hello,
Make sure your browser is maximized and check the quiz availability in your region

**Comment by inverspe** [comment]
> Hello, Make sure your browser is maximized and check the quiz availability in your region
is it possible for you to make sure that headless mode go into fullscreen? (I know you can't see the gui)

**Comment by farshadz1997** [comment]
I configed it to open browser in full size

**Comment by pro7777777777** [comment]
Yes the browser is in full size and the msn quiz available because I have done it manually before.

**Comment by farshadz1997** [comment]
Check manually msn quiz when bot is trying to complete it, make sure everything is fine, and also check is your account logged in or not

**Comment by farshadz1997** [comment]
Hello, get the latest commit and try again

**Comment by JittoJoseph** [comment]
[MSN GAME] Failed to complete MSN game!
